x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/integration/DocumentLevelSecurityTests.java

```diff
@@ -1400,13 +1402,15 @@ public void testProfile() throws Exception {
         assertNoFailures(response);
 
         assertThat(response.getProfileResults().size(), equalTo(1));
-        SearchProfileQueryPhaseResult shardResult = response.getProfileResults().get(response.getProfileResults().keySet().toArray()[0]);
+        SearchProfileShardResult shardResult = response.getProfileResults().get(response.getProfileResults().keySet().toArray()[0]);
         assertThat(shardResult.getQueryProfileResults().size(), equalTo(1));
         QueryProfileShardResult queryProfileShardResult = shardResult.getQueryProfileResults().get(0);
         assertThat(queryProfileShardResult.getQueryResults().size(), equalTo(1));
         logger.info("queryProfileShardResult=" + Strings.toString(queryProfileShardResult));
-//        ProfileResult profileResult = queryProfileShardResult.getQueryResults().get(0);
-//        assertThat(profileResult.getLuceneDescription(), equalTo("(other_field:value)^0.8"));
+        assertThat(
+            queryProfileShardResult.getQueryResults().stream().map(ProfileResult::getLuceneDescription).sorted().collect(toList()),
+            equalTo(List.of("(other_field:value)^0.8"))
+        );
```

Comment:
This had been commented out for years. I'm not sure why. It passes quite consistently for me. I think its worth uncommenting it and watching.

Reply:
Hopefully folks'll git-blame it and come for me. But yeah, I'll ping them before merging.